airbyte-integrations/connectors/source-google-drive/integration_tests/spec.json

```diff
@@ -0,0 +1,395 @@
+                              "title": "Header Definition Type",
+                              "default": "From CSV",
+                              "const": "From CSV",
+                              "type": "string"
+                            }
+                          }
+                        },
+                        {
+                          "title": "Autogenerated",
+                          "type": "object",
+                          "properties": {
+                            "header_definition_type": {
+                              "title": "Header Definition Type",
+                              "default": "Autogenerated",
+                              "const": "Autogenerated",
+                              "type": "string"
+                            }
+                          }
+                        },
+                        {
+                          "title": "User Provided",
+                          "type": "object",
+                          "properties": {
+                            "header_definition_type": {
+                              "title": "Header Definition Type",
+                              "default": "User Provided",
+                              "const": "User Provided",
+                              "type": "string"
+                            },
+                            "column_names": {
+                              "title": "Column Names",
+                              "description": "The column names that will be used while emitting the CSV records",
+                              "type": "array",
+                              "items": {
+                                "type": "string"
+                              }
+                            }
+                          },
+                          "required": ["column_names"]
+                        }
+                      ],
+                      "type": "object"
+                    },
+                    "true_values": {
+                      "title": "True Values",
+                      "description": "A set of case-sensitive strings that should be interpreted as true values.",
+                      "default": ["y", "yes", "t", "true", "on", "1"],
+                      "type": "array",
+                      "items": {
+                        "type": "string"
+                      },
+                      "uniqueItems": true
+                    },
+                    "false_values": {
+                      "title": "False Values",
+                      "description": "A set of case-sensitive strings that should be interpreted as false values.",
+                      "default": ["n", "no", "f", "false", "off", "0"],
+                      "type": "array",
+                      "items": {
+                        "type": "string"
+                      },
+                      "uniqueItems": true
+                    },
+                    "inference_type": {
```

Comment:
This is a legacy option that's just for existing S3 connections so can be deleted.